deepchecks/nlp/checks/model_evaluation/train_test_performance.py

```diff
@@ -33,6 +33,17 @@ class TrainTestPerformance(TrainTestPerformanceAbstract, TrainTestCheck):
     scorers: Union[Mapping[str, Union[str, Callable]], List[str]], default: None
         Scorers to override the default scorers, find more about the supported formats at
         https://docs.deepchecks.com/stable/user-guide/general/metrics_guide.html
+    min_samples: int, default: 30
```

Comment:
hmm, maybe 10? for score per class I think its a bit harsh (sometime we will be evaluated on a small dataset)

Reply:
It's about significance as well. you'll have a drop of 10% for a change in one sample out of 10. I think 10 is too low, and 30 is a common min sample size.

Reply:
I mean, in general, if my metrics reduced on such a small class, I also wouldn't care...